docs/conf.py

```diff
@@ -61,6 +61,10 @@
     ("py:class", "SimTime"),
     ("py:class", "immutables._map.Map"),
     ("py:class", "_asyncio.Task"),
+    ("py:class", "tickit.devices.cryostream.status.Status"),
+    ("py:class", "tickit.devices.cryostream.status.ExtendedStatus"),
```

Comment:
Am I correct in thinking this is due to autosphinx playing up when trying to point to `Enum`s?